packages/trading-widget/src/core-kit/hooks/trading/synthetix-v3/use-synthetix-v3-oracles-update.ts

```diff
@@ -30,23 +34,28 @@ export const useSynthetixV3OraclesUpdate = ({
   const { sendTransaction } = useSendOraclesUpdateTransaction({
     chainId,
   })
-  const { data: oraclesUpdateTransactionData, isFetching } =
-    useOraclesUpdateTransactionData(
-      {
-        publicClient,
-        account,
-        chainId,
-        vaultAddress,
-      },
-      {
-        refetchInterval: EXTREMELY_SHORT_POLLING_INTERVAL,
-        retry: DEFAULT_RETRIES_NUMBER,
-        enabled: isSynthetixVault && !disabled,
-      },
-    )
+  const {
+    data: {
+      isOracleDataUpdateNeeded,
+      prependedTxns,
+    } = DEFAULT_ORACLES_DATA_RESPONSE,
+    isFetching,
+  } = useOraclesUpdateTransactionData(
+    {
+      publicClient,
+      account,
+      chainId,
+      vaultAddress,
+    },
+    {
+      refetchInterval: EXTREMELY_SHORT_POLLING_INTERVAL,
+      retry: DEFAULT_RETRIES_NUMBER,
```

Comment:
just to confirm, so this is the retry setting here for running the buildPrependedTxs function. yes, we should have some retries here.

Reply:
Yes, this retry settings is for running `buildPrependedTxs `